src/instructlab/config.py

```diff
@@ -38,6 +38,17 @@
 # use spawn start method, fork is not thread-safe
 DEFAULT_MULTIPROCESSING_START_METHOD = "spawn"
 
+# When otherwise unknown, ilab uses this as the default family
+DEFAULT_MODEL_FAMILY = "merlinte"
```

Comment:
```suggestion
DEFAULT_MODEL_FAMILY = "merlinite"
```

Did you mean `merlinite`?

Reply:
Yes - that explains the test failure, too. Thank you!